### PR TITLE
Update dependencies in Application Connector components

### PR DESCRIPTION
**Pull request by Szymongib**
**Description**

Changes proposed in this pull request:

- Update dependencies in:
    - Application Connectivity Validator
    - Application Registry
    - Application Gateway
    - Application Gateway Tests

**Related issue(s)**

### CODE REVIEW

File: tests/application-gateway-tests/go.mod [hunk 1]
```diff
@@ -5,16 +5,15 @@ go 1.14
...
-	gopkg.in/yaml.v2 v2.2.8 // indirect
-	k8s.io/api v0.17.2
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
+	k8s.io/api v0.18.4
+	k8s.io/apimachinery v0.18.4
+	k8s.io/client-go v0.18.4
 )
+
+replace github.com/kyma-project/kyma/components/application-gateway => github.com/Szymongib/kyma/components/application-gateway v0.0.0-20200902080656-573e032dd983
```
Comment (rafalpotempa) [inline tests/application-gateway-tests/go.mod:19]: I think it's an artefact

**Review by rafalpotempa — DISMISSED** [review]

File: tests/application-gateway-tests/go.mod (hunk 1 above)
Comment (Szymongib) [reply tests/application-gateway-tests/go.mod:19]: It is needed as master version of Gateway uses an older version of the Application Operator, which actually uses the old version of the service-catalog client.

**Review by majakurcius — DISMISSED** [review]
🚀

**Review by rafalpotempa — APPROVED** [review]

**Pull request merged**